app/src/main/java/com/example/myapplication/ManaListItem.java

```diff
@@ -0,0 +1,40 @@
+package com.example.myapplication;
+
+public class ManaListItem {
```

Comment:
What is the purpose of this class? Is it for the ItemPageViewer ?

Reply:
Exactly :+1: